modules/tensor_mechanics/tests/mohr_coulomb/tests

```diff
@@ -126,6 +133,13 @@
     rel_err = 1.0E-5
     abs_zero = 1.0E-3
   [../]
+  [./uni_axial1_small_strain]
+    type = 'CSVDiff'
+    input = 'uni_axial1_small_strain.i'
+    csvdiff = 'uni_axial1.csv'
+    rel_err = 1.0E-5
+    abs_zero = 1.0E-3
```

Comment:
There is.  Basically, the misfit from the original residual (before the preset has been applied) is the "true" residual.... it's what we want to reduce.  We then apply the preset as a good "guess" at the solution... and the residual will drop a lot.  If you tried to converge from the "dropped" residual you could run out of precision (plus it doesn't represent how well you fit your true problem).

Let's run a scenario.  Let's say you have an initial residual of 1e-4.  After applying the preset it drops to 1e-8.  If you want to try to converge 8 orders of magnitude that would be 1e-12 beyond the initial residual... which is doable.  If you try to go 8 orders of magnitude beyond 1e-8 you're now trying to get to 1e-16.... which probably isn't going to happen... and that represents a _12_ order of magnitude drop from the true initial residual... WAY over solving.

You can ask @bwspenc for more information.